packages/tools/devtools/devtools/src/index.ts

```diff
@@ -173,18 +162,14 @@ export function initializeDevtools(props: DevtoolsProps): IDevtools {
  */
 function mapContainerProps(
 	containerProps: ContainerDevtoolsProps,
-): ContainerDevtoolsPropsBase | undefined {
+): ContainerDevtoolsPropsBase {
 	const { container, containerKey } = containerProps;
-	const fluidContainer = container as { INTERNAL_CONTAINER_DO_NOT_USE?: () => IContainer };
-
-	if (fluidContainer.INTERNAL_CONTAINER_DO_NOT_USE === undefined) {
-		console.error("Missing Container accessor on FluidContainer.");
-		return undefined;
+	if (!isInternalFluidContainer(container)) {
+		throw new TypeError("Container is not required FluidContainer.");
```

Comment:
Since theoretically end-users will see this, we should be a bit more descriptive here. Note that the `IFluidContainer` implementation was unrecognized, and only Containers generated by the Fluid-Framework are supported.